bin/k8s-deploy

```diff
@@ -152,6 +152,24 @@ do
     echo "Updating ${DEPLOYMENT_FILE}-${CI_SHA1}"
     DEPLOYMENT_FILE=${DEPLOYMENT_FILE}-${CI_SHA1}
   fi
+  if [[ $( grep -i replicas $DEPLOYMENT_FILE | awk -F: {'print $2'} | grep -i hpa ) ]]
+  then
+    REPLICA_COUNT=$( kubectl get deployment kube-flask --namespace=staging |
```

Comment:
Looks like this is hardcoded to your test deployment and namespace. Am I missing something?

Reply:
do'h, thanks for catching that, copy paste error

Reply:
Ok, fixed and tested